package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "8-9july",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@joi/date": "^2.1.0",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "joi": "^17.9.2",
+    "uuid": "^9.0.0"
```

Comment:
nodejs already has a crypto which has uuid function

Reply:
I studied about crypto.getrandomuuid() and I will remove uuid library and use crypto.getrandomuuid() in its place